refkit-init-build-env

```diff
@@ -30,50 +30,23 @@ if [ -n "$BASH_SOURCE" ]; then
 elif [ -n "$ZSH_NAME" ]; then
     THIS_SCRIPT=$0
 else
-    THIS_SCRIPT="$(pwd)/oe-init-build-env"
-fi
-if [ -n "$BBSERVER" ]; then
-    unset BBSERVER
+    THIS_SCRIPT="$(pwd)/refkit-init-build-env"
 fi
 
-if [ -z "$ZSH_NAME" ] && [ "$0" = "$THIS_SCRIPT" ]; then
-    echo "Error: This script needs to be sourced. Please run as '. $THIS_SCRIPT'"
-    exit 1
-fi
+THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
+THIS_SCRIPT_DIR=$(readlink -f "$THIS_SCRIPT_DIR")
 
-if [ -z "$TEMPLATECONF" ]; then
-    THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
-    THIS_SCRIPT_DIR=$(readlink -f "$THIS_SCRIPT_DIR")
-    if [ -f "$THIS_SCRIPT_DIR/.templateconf" ]; then
-        source $THIS_SCRIPT_DIR/.templateconf
-    elif [ -d "$THIS_SCRIPT_DIR/.template" ]; then
-        TEMPLATECONF="$THIS_SCRIPT_DIR/.template"
-    fi
+if [ -z "$OEROOT" ]; then
+    OEROOT="$THIS_SCRIPT_DIR"/openembedded-core
 fi
```

Comment:
This looks a lot saner than before.

I just wonder whether OEROOT and BITBAKEDIR are allowed to be set before sourcing refkit-init-build-env. The "if" check seems to imply that this is allowed. When used in this way, is the current .templateconf going to work? It's falls back to a relative path that won't be correct.

It fails for me, as expected:

    $ export OEROOT=/fast/work/openembedded-core
    $ . /fast/work/intel-iot-refkit/refkit-init-build-env /tmp/test
    Error: TEMPLATECONF value points to nonexistent directory '../meta-refkit/conf'